staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -407,9 +408,6 @@ func (o *ApplyOptions) Validate() error {
 				return fmt.Errorf("--selector is incompatible with --applyset")
 			} else if len(o.PruneResources) > 0 {
 				return fmt.Errorf("--prune-allowlist is incompatible with --applyset")
-			} else {
-				// TODO: remove this once ApplySet implementation is complete
-				return fmt.Errorf("--applyset is not yet supported")
```

Comment:
suggestion: if your PR goes in before mine, replace with with a logged warning that the feature does not work correctly